Oxide.Core/Localization.cs

```diff
@@ -70,7 +70,8 @@ public static class Localization
                 ["PlayerRemovedFromGroup"] = "Player '{0}' removed from group '{1}'",
                 ["PlayersFound"] = "Multiple players were found, please specify: {0}",
                 ["Version"] = "Server is running [#ffb658]Oxide {0}[/#] and [#ee715c]{1} {2} ({3})[/#]",
-                ["YouAreNotAdmin"] = "You are not an admin"
+                ["YouAreNotAdmin"] = "You are not an admin",
+                ["OxideSaved"] = "Saving Oxide data..."
```

Comment:
Should not contain "Oxide" in the key, would be better as "SavingData" and "Saving Oxide data..." and likely be good to add a "SavedData" key as well for post save.